### PR TITLE
BXC-3737 - Allow descriptions to be added to files in single file works

### PR DESCRIPTION
**Pull request by bbpennel**
https://jira.lib.unc.edu/browse/BXC-3737

* Allow descriptions to be added to files in single file works

### CODE REVIEW

File: src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java [hunk 1]
```diff
@@ -298,7 +294,8 @@ protected void generateWork() throws IOException {
 
         protected List<PID> addChildObjects() throws IOException {
             SourceFileMapping sourceMapping = getSourceFileMapping(cdmId);
-            return Collections.singletonList(addFileObject(cdmId, cdmCreated, sourceMapping));
+            return Collections.singletonList(addFileObject(cdmId, cdmId + "/original_file",
```
Comment (sharonluong) [inline src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java:297]: so here the `cdmId + "/original_file"` specifies the file in the work?

File: src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java [hunk 2]
```diff
@@ -229,11 +229,7 @@ protected List<PID> addChildObjects() throws IOException {
                     String cdmCreated = rs.getString(2) + "T00:00:00.000Z";
 
                     SourceFileMapping sourceMapping = getSourceFileMapping(cdmId);
-                    PID filePid = addFileObject(cdmId, cdmCreated, sourceMapping);
-                    var childDescPath = getDescriptionPath(cdmId, true);
-                    if (childDescPath != null) {
-                        copyDescriptionToSip(filePid, childDescPath);
-                    }
+                    PID filePid = addFileObject(cdmId, cdmId, cdmCreated, sourceMapping);
```
[the remark this answers is not in the document]
Comment (bbpennel) [reply src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java]: I've split things up so it doesn't have to pass double ids to this method anymore, instead there's a separate call to add the description, hopefully this helps make it clearer what the purpose of the ids is.

File: src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java (hunk 1 above)
Comment (bbpennel) [reply src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java]: I've added a comment to describe the reason for this suffix

File: src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharonluong) [reply src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java]: ah that does make more sense

**Review by sharonluong — APPROVED** [review]

**Pull request merged**